README.md

```diff
@@ -177,6 +177,24 @@ See [samples/java/jdbc](samples/java/jdbc) for a small sample application that a
 PGAdapter as a compile-time dependency and runs it together with the main application.
 
 ## Emulator
+A pre-built Docker image that contains both PGAdapter and the Spanner Emulator can be started with
+these commands:
+
+```shell
+docker pull gcr.io/cloud-spanner-pg-adapter/pgadapter-emulator
+docker run \
+  -d -p 5432:5432 \
+  gcr.io/cloud-spanner-pg-adapter/pgadapter-emulator
+sleep 2
```

Comment:
Is this necessary?

Reply:
Yeah, to be sure that users who copy-paste this exact script don't run into anything, then we need something here. The reason is that:

1. PGAdapter needs a bit of time to start up (some ~hundreds of milliseconds).
2. The PostgreSQL TCP port is only available once PGAdapter has started.
3. However, as Docker is started in detached mode in this case, the `docker run` command will finish directly.
4. That again means that the `psql` command following will fail with a somewhat random error if it tries to connect before PGAdapter has started up properly.

One alternative could have been add some kind of polling loop that checks whether port 5432 is available, but this feels 'safer' that it will work on all hosts.